pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -70,6 +70,9 @@ type ControllerConfigSpec struct {
 
 	// TODO: Use string for CA data
 
+	// managed Kubelet to API Server Cert... Rotated automatically
```

Comment:
This comment line has to start with `kubeAPIServerServingCAData ...` (ie. the json name)

Reply:
@LorbusChris do you know if the TODO in line 71 above tracked anywhere?

Reply:
Not one that I'm aware of. There are a few TODOs in types.go (mostly for a future v2 of the api). I'd say we'll decide what to do with these/open issues once we move the apis package over to openshift/api (either here or there)

Reply:
sounds good, just dont want to lose track of them

Reply:
@LorbusChris fixed the comment!